openhantek/src/configdialog/DsoConfigProbePage.cpp

```diff
@@ -0,0 +1,93 @@
+//
```

Comment:
I forgot to add this information to the contribution guideline, but please use the license header line

Reply:
I'l modify it!